Week-1/assignment-1.2/style.css

```diff
@@ -0,0 +1,119 @@
+* {
+   font-family: 'League Spartan', sans-serif;
+   overflow: hidden;
+}
+
+#asideicons {
+   list-style-type: none;
+   margin: 3px;
+   padding: 0;
+   width: 3%;
+   background-color: #f1f1f1;
+   position: fixed;
+   height: 100%;
+}
+
+li {
+   display: block;
+   color: #000;
+   padding: 35px 16px;
+   text-decoration: none;
+}
+
+nav {
+   margin-left: 15px;
+   height: 315px;
```

Comment:
do not give fixed height like this. instead keep it fluid and according to content.